packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts

```diff
@@ -0,0 +1,42 @@
+import { PayloadAction } from '@reduxjs/toolkit'
+import {
+  CertFieldsTypes,
+  getCertFieldValue,
+  loadCertificate,
+  parseCertificate,
+  keyFromCertificate
+} from '@quiet/identity'
+import { put, select, call } from 'typed-redux-saga'
+import { messagesActions } from '../../messages/messages.slice'
+import { WriteMessagePayload } from '../../messages/messages.types'
+import { publicChannelsActions } from '../publicChannels.slice'
+import { usersSelectors } from '../../users/users.selectors'
+import { communitiesSelectors } from '../../communities/communities.selectors'
+
+import { MAIN_CHANNEL } from '../../../constants'
+
+export function* sendNewUserInfoMessageSaga(
+  action: PayloadAction<ReturnType<typeof publicChannelsActions.sendNewUserInfoMessage>['payload']>
+): Generator {
+  const community = yield* select(communitiesSelectors.currentCommunity)
+  const isOwner = community.CA
+
+  if (!isOwner) return
```

Comment:
why no `if (!community.CA) return` ?

Reply:
maybe we should check it before action call and do not call this saga when its not an owner

Reply:
I'd leave all logic in sagas and keep eventChannel clean and have just one responsibility - dispatch actions

Reply:
got it